src/components/form/Form.js

```diff
@@ -0,0 +1,204 @@
+            email: '',
+            request: ''
+        })
+        setErrors({
+            name: '',
+            email: ''
+        })
+        setIsValid(false);
+        setNameValid(true);
+        setEmailValid(true);
+    }
+
+    // ___________________reset inputs when popup is closed
+    useEffect(() => {
+        resetInputs();
+    }, [props.open]);
+
+    // ____________________________close popup
+    const closePopup = () => {
+        props.closePopup();
+    }
+
+    // ___________________________________ Google form data
+    const formName = 'entry.1259736291';
+
+    const formEmail = 'entry.1503900032';
+
+    const formRequest = 'entry.1763891937';
+
+    const formURL = 'https://docs.google.com/forms/d/18EfQ-4I_6IffAlGivd0wxGbebGZ1REUGuVuJe4_YvgE/formResponse';
+
+    let submittedForm = false;
+    let hiddenFrameLoaded = false;
+
+    function handleHiddenFrameLoaded() {
+        if (hiddenFrameLoaded) {
+            submittedForm = true;
+        } else {
+            hiddenFrameLoaded = true;
+        }
+    }
+
+    return (
+        <Overlay open={props.open}>
+            <Wrapper>
+                <Exit
+                    src={exit}
+                    onClick={closePopup} />
+                <Title>
+                    Leave your contacts and request,
+                    and our manager will reach out to you to
+                    establish our futher parntership!
+                </Title>
+                <Iframe
+                    title='hidden_iframe'
+                    name='hidden_iframe'
+                    id='hidden_iframe'
+                    onload={handleHiddenFrameLoaded} />
+                <ConnectForm
+                    ref={formRef}
+                    onChange={checkFormValidity}
+                    target='hidden_iframe'
+                    action={formURL}
+                    method='POST'
+                    onSubmit={closePopup}>
+                    <InputWrap>
+                        <Label
+                            $valid={nameValid}>
+                            Personal or company name *
+                            {!nameValid && errorMessage}
+                        </Label>
+                        <Input
+                            $valid={nameValid}
+                            type='text'
+                            name={formName}
+                            id='name'
+                            required
+                            onChange={handleChange}
+                            onBlur={updateErrors}
+                            value={inputs.name} />
+                    </InputWrap>
+                    <InputWrap>
+                        <Label
+                            $valid={emailValid}>
+                            Email *
+                            {!emailValid && errorMessage}
+                        </Label>
+                        <Input
+                            $valid={emailValid}
+                            type='text'
```

Comment:
try changing the type to 'email'. Right now you can type a single letter and it's considered valid

Reply:
I think this is an improvement but it's considered valid when you enter an incomplete email like "riley@g"... Not sure what the issue is though, just thought I'd mention it.